src/pages/superadmin/tableComponent/index.tsx

```diff
@@ -335,6 +375,92 @@ export const MyTable = ({
           </BountyHeader>
           <Options>
             <FlexDiv>
+              <EuiPopover
+                button={
+                  <StatusContainer onClick={onProviderButtonClick} color={color}>
+                    <EuiText
+                      className="statusText"
+                      style={{
+                        color: isProviderPopoverOpen ? color.grayish.G10 : ''
+                      }}
+                    >
+                      Provider:
+                    </EuiText>
+                    <div className="filterStatusIconContainer">
+                      <MaterialIcon
+                        className="materialStatusIcon"
+                        icon={`${
+                          isProviderPopoverOpen ? 'keyboard_arrow_up' : 'keyboard_arrow_down'
+                        }`}
+                        style={{
+                          color: isProviderPopoverOpen ? color.grayish.G10 : ''
+                        }}
+                      />
+                    </div>
+                  </StatusContainer>
+                }
+                panelStyle={{
+                  border: 'none',
+                  boxShadow: `0px 1px 20px ${color.black90}`,
+                  background: `${color.pureWhite}`,
+                  borderRadius: '0px 0px 6px 6px',
+                  marginTop: '0px',
+                  marginLeft: '20px'
+                }}
+                isOpen={isProviderPopoverOpen}
+                closePopover={closeProviderPopover}
+                panelPaddingSize="none"
+                anchorPosition="downLeft"
+              >
+                <ProviderContainer>
+                  <ProvidersListContainer>
+                    {providers && providers.length > 0 ? (
```

Comment:
Add an intersection observer to load more providers on scroll when the provider length is more than 5

Reply:
Hi @elraphty! Why is there a need for the Intersection Observer to load more providers on scroll when we are already receiving all provider data from the metrics/bounties endpoint call? We are not making another call for additional providers unless triggered by changes in the date filter, for example. So, my point is, since we already have all providers, it would be more appropriate to implement the Intersection Observer if we need to call the API to get more providers. Could you please correct me if I am wrong or guide me?